src/Order.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
+
+import RestaurantButton from "./RestaurantButton";
+
+const Order = props => {
+  const [orders, setOrders] = useState(0);
+
+  const orderOne = () => {
+    return setOrders(orders + 1);
```

Comment:
This is great! Well done for implementing this!

Do you think the `return` statement is necessary?